### DISCUSSION TITLE
Zksync beta node wallet link

### DISCUSSION THREAD

**Discussion by cyrptoburrys** [zkSync-Community-Hub/zksync-developers#591] [General] [answered]
Been running the (zksync beta node, mainnet one ) 
And i im wondering how to link my wallet to the node ? 
Also how to link my node to github ? 

But first let me show u where im at with the node : 

- Docker Containers: The zkSync node container is up and running.
-Health Check: The response from the health endpoint (curl http://localhost:3081/health) shows that the status is "ready" and "is_synced": true.
-API Check: The JSON-RPC call to eth_syncing returns false, indicating that the node is fully synced.![image](https://github.com/zkSync-Community-Hub/zksync-developers/assets/136279935/0fd08d51-c14d-4eef-952e-157d78f4e864)

So im here Right now , what to do next and how to link my wallet 
And to link it to my github 

Note: its the first time for me to run a node i used chatgpt for help 
So im not expert , 

For more details about my node see this: root@vmi1913714:~# docker ps
CONTAINER ID   IMAGE                                  COMMAND                  CREATED      STATUS                PORTS                                                                    NAMES
6317437c0b73   matterlabs/external-node:2.0-v24.6.0   "sh /usr/bin/entrypo…"   4 days ago   Up 4 days             127.0.0.1:3060-3061->3060-3061/tcp, 127.0.0.1:3081->3081/tcp, 3322/tcp   docker-compose-examples-external-node-1
0e2b4caa23a3   postgres:14                            "docker-entrypoint.s…"   4 days ago   Up 4 days (healthy)   5430/tcp, 5432/tcp                                                       docker-compose-examples-postgres-1
99a6a91367f5   grafana/grafana:9.3.6                  "/run.sh"                4 days ago   Up 4 days             127.0.0.1:3000->3000/tcp                                                 docker-compose-examples-grafana-1
6c4b1f6a2215   prom/prometheus:v2.35.0                "/bin/prometheus --c…"   4 days ago   Up 4 days             9090/tcp                                                                 docker-compose-examples-prometheus-1
root@vmi1913714:~# curl http://localhost:3081/health
{"status":"ready","components":{"http_api":{"status":"ready"},"batch_status_updater":{"status":"ready","details":{"last_committed_l1_batch":485423,"last_executed_l1_batch":485326,"last_proven_l1_batch":485413}},"sync_state":{"status":"ready","details":{"is_synced":true,"local_block":36510254,"main_node_block":36510254}},"reorg_detector":{"status":"ready","details":{"last_correct_l1_batch":485423,"last_correct_l2_block":36510253}},"prometheus_exporter":{"status":"ready"},"tree":{"status":"ready","details":{"leaf_count":332204452,"min_l1_batch_number":484823,"mode":"lightweight","next_l1_batch_number":485424,"root_hash":"0x87bdaa026293f866ea247b677507cafa81e9e752d0b0b9245bad8ce6e9f7ba96","stage":"main_loop"}},"commitment_generator":{"status":"ready","details":{"l1_batch_number":485423}},"connection_pool":{"status":"ready","details":{"max_size":10,"pool_size":10}},"ws_api":{"status":"ready"},"consistency_checker":{"status":"ready","details":{"first_checked_batch":484824,"last_checked_batch":485423}},"main_node_http_rpc":{"status":"ready"},"ethereum_http_rpc":{"status":"ready"},"snapshot_recovery":{"status":"ready","details":{"factory_deps_recovered":true,"snapshot_l1_batch":484823,"snapshot_l2_block":36154870,"storage_logs_chunk_count":331,"storage_logs_chunks_left_to_process":0,"tokens_recovered":true}}}}root@vmi1913714:~# curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"eth_syncing","params":[],"id":1}' http://localhost:3060/api/v0.1
{"jsonrpc":"2.0","result":false,"id":1}

**Comment by bxpana** [comment, marked as answer]
You can change the RPC endpoint in your wallet and be able to use it. What do you mean by connect it to your Github? What do you want to do with that?
